py/desispec/coaddition.py

```diff
@@ -251,41 +257,64 @@ def coadd_fibermap(fibermap, onetile=False):
         good_coadds = np.bitwise_not( nonamp_fiberstatus_flagged | allamps_flagged )
         tfmap['COADD_NUMEXP'][i] = np.count_nonzero(good_coadds)
 
+        #- For FIBER_RA/DEC quantities, only average over good coordinates
+        #  There is a bug that some "missing" coordinates were set to FIBER_RA=FIBER_DEC=0
+        #  (we are assuming there are not valid targets at exactly 0,0; only missing coords)
+        if 'FIBER_RA' in fibermap.colnames:
+            good_coords = (fibermap['FIBER_RA'][jj]!=0)|(fibermap['FIBER_RA'][jj]!=0)
```

Comment:
typo: FIBER_RA listed twice instead of FIBER_RA and FIBER_DEC

Reply:
Fixed.